prodtests/full-system-test/calib-workflow.sh

```diff
@@ -16,6 +16,12 @@ if workflow_has_parameters CALIB_LOCAL_INTEGRATED_AGGREGATOR CALIB_PROXIES; then
   exit 2
 fi
 
+if [[ -z ${NSLICESPERTF:-} ]]; then
+  NSLICESPERTF=$(echo "scale=2;$NHBPERTF/128*11" | bc)
```

Comment:
could you make this variable name more specific? like TPC_IDCCALIB_SLICESPERTF?

Reply:
sure